Source/SwiftLintFramework/Rules/Lint/HashValueOverflowRule.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import SourceKittenFramework
+
+public struct HashValueOverflowRule: ASTRule, OptInRule, ConfigurationProviderRule, AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "hash_value_overflow",
```

Comment:
I'd rename this function and update the description, because IMO the rule is now more general than just guarding against overflows: the new hash function should be preferred in general.

Reply:
Would `HashFunctionRule` be something that is OK?

Reply:
sounds good to me 👍